### ISSUE TITLE
move command

### ISSUE THREAD

**Issue by proofit404** [emre/storm#99]
I's would be great if there were following command:
```bash
storm move old_name new_name
```
Currently I new clone old_name into new_name and manually delete old_name entry.

Regards, Artem.

**Comment by emre** [comment]
should be simple touch, will be added with the new version

I think, a flag will do the job. something like $ storm clone --remove-original

**Comment by proofit404** [comment]
In my opinion it must be something like `cp` and `mv` shell commands. But flag is also fine.

**Comment by emre** [comment]
shipped with 0.6.6.
```bash
$ storm move host new_host
```

you can use aliases for bash-like commands. (config file: ~/.stormssh/config)
```javascript
{
    "aliases": {
        "add": ["create", "touch"],
        "delete": ["rm"],
        "move": ["mv"],
	 	"clone": ["cp"]
    }

}
```

**Comment by proofit404** [comment]
Thanks.
